control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -591,3 +591,45 @@ impl From<VolumeSpec> for models::VolumeSpec {
         )
     }
 }
+
+/// VolumeGroupId type.
+pub type VolumeGroupId = String;
+
+/// In memory representation of volume group.
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct VolumeGroupSpec {
+    /// Name of the volume group.
+    name: VolumeGroupId,
```

Comment:
should this be id and not name? If so also fixup getter method,.

Reply:
I initially thought of id. But its not really an id right? by id I mean uuid format. What do you think, is id a better term here?

Reply:
Id is just an Id, UUID is different

Reply:
I see, alright. Will change the names to id.

Reply:
Hmm we might want a uuid indeed, in fact I'm wondering if we should have a uuid though it would make it harder for csi layer since the vg id is created based on funky pvc and ss logic.

Reply:
Yeah. Right.